Makefile

```diff
@@ -62,6 +62,7 @@ clean:
 
 test:
 	export PATH=$(PATH) CC=$(CC) CXX=$(CXX) && bazel $(BAZEL_STARTUP_ARGS) test $(BAZEL_BUILD_ARGS) $(BAZEL_CONFIG_DEV) $(BAZEL_TARGETS)
+	GO111MODULE=on go test ./...
```

Comment:
just so I understand: we can't use `bazel test` to run the golang tests with `GO111MODULE=on` ?

Reply:
No, it has nothing to do with go mod. I am just too lazy to add bazel rules for the go test part :). If it does not look good to use go native build I could try to add bazel rules for golang code.

Reply:
We can add a bazel build eventually. Envoy already comes with all go set-up needed. Just no reason to force it yet.